.travis.yml

```diff
@@ -45,6 +45,7 @@ before_script:
 
 
 script:
+  - bash -c 'shopt -s globstar nullglob; shellcheck **/*.{sh,ksh,bash}'
```

Comment:
Can we add this to the `Makefile` instead? I'd like to be able to run `make shellcheck` from my laptop for local development. Also, the `vendor` directory should be ignored.

Reply:
Suggested changes are done.